server/keyspace_service.go

```diff
@@ -141,3 +141,21 @@ func (s *KeyspaceServer) UpdateKeyspaceState(_ context.Context, request *keyspac
 		Keyspace: meta,
 	}, nil
 }
+
+// GetAllKeyspaces get all keyspace's metadata.
+func (s *KeyspaceServer) GetAllKeyspaces(_ context.Context, request *keyspacepb.GetAllKeyspacesRequest) (*keyspacepb.GetAllKeyspacesResponse, error) {
+	if err := s.validateRequest(request.GetHeader()); err != nil {
+		return nil, err
+	}
+
+	manager := s.GetKeyspaceManager()
+	keyspaces, err := manager.LoadRangeKeyspace(request.StartId, int(request.Limit))
```

Comment:
How often will this interface be called, and will it cause additional performance issues if all keyspace is loaded from etcd each time? How about using watch loop?